src/ServiceFabric/ServiceFabric/help/Add-AzServiceFabricManagedClusterNetworkSecurityRule.md

```diff
@@ -0,0 +1,377 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll-Help.xml
+Module Name: Az.ServiceFabric
+online version: https://learn.microsoft.com/powershell/module/az.servicefabric/add-azservicefabricmanagedclusternetworksecurityrule
+schema: 2.0.0
+---
+
+# Add-AzServiceFabricManagedClusterNetworkSecurityRule
+
+## SYNOPSIS
+Add network security rule to cluster resource.
+
+## SYNTAX
+
+### ByObj (Default)
+```
+Add-AzServiceFabricManagedClusterNetworkSecurityRule 
+[-InputObject] <PSManagedCluster>
+[-Name] <String>
+[-Access] <String>
+[-Description] <String>
+[-DestinationAddressPrefix] <String[]>
+[-DestinationPortRange] <String[]>
+[-Direction] <String>
```

Comment:
you need to regenerate the help notes now that youve modified the cmdlets

Reply:
when you regenerate you'll also have to update the examples to use the new enum types. itll be something like [Microsoft.Azure.Commands.ServiceFabric.Models.Protocol]::http

Reply:
Examples updated